src/components/TooltipNew/GuideTooltip.md

```diff
@@ -0,0 +1,139 @@
+<h3>GuideTooltip</h3>
```

Comment:
Docs seem incomplete. Didn't Justyna write any description of this new component?

Reply:
I added more docs based on what Justyna provided